src/main/kotlin/datamaintain/Script.kt

```diff
@@ -4,11 +4,13 @@ package datamaintain
 interface Script {
     val name: String
     val checksum: String
+    val tags: Set<Tag>
```

Comment:
I'm not sure about having ``tags`` in ``Script``. I don't think all scripts have tags, see further comments.